lms/static/sass/elements/_controls.scss

```diff
@@ -1,6 +1,399 @@
 // lms - elements - controls
 // ====================
 
+// in this document:
+// --------------------
+// +notes
+// +button - bourbon add-on
+// +existing/misc buttons (CLEANUP!)
+
+
+// +notes:
+// --------------------
+// * this contains all utility-based/focused mixins that can be used throughout the platform.
+
+
+// +button - bourbon add-on
```

Comment:
NOTE: Since the LMS relies on the Bourbon ``button()`` mixin so much, I've taken its original syntax, right before it was deprecated, and copied it here. To make sure that we 1) know that it is bourbon-based, and 2) this crutch is removed as soon as we revise buttons, I've given the mixin's name a bourbon reference (``button-bourbon``).

If this way of handling the dependency is an issue, I'm happy to hear efficient alternatives (I spent a night or two trying to recreate all aspects of the button in our own edX-created mixin/extends, but I couldn't promise the same rendering across every case that called the original mixin).

Also, If the location or the name are an issue, I'm happy to take suggestions on any improvements.